src/components/user-list-test.html

```diff
@@ -0,0 +1,49 @@
+<link rel="import" href="../../node_modules/@banno/polymer/polymer.js">
+<link rel="import" href="../../node_modules/@banno/polymer/lib/elements/dom-repeat.js">
+<link rel="import" href="./user-profile.html">
+
+<dom-module id="user-list">
+  <style>
+  </style>
+  <template>
+    <h2> User list: </h2>
+    <dom-repeat items="{{users}}">
```

Comment:
I'm sure this works, however, you should get used to doing it this way:
```
<template is="dom-repeat" items="[[users]]>
  <div>
    <user-profile ...></user-profile>
  </div>
</template>
```
Beyond "it's what we do", there's a good reason for using the `template is="..."` syntax over `dom-repeat`. At the moment it escapes me, though.

Reply:
Also needs to be one-way bound, like the properties on `user-profile`